lib/money_column/active_record_hooks.rb

```diff
@@ -37,11 +37,13 @@ def money_column(*columns, currency_column: nil, currency: nil, currency_read_on
         else
           raise ArgumentError, 'must set one of :currency_column or :currency options'
         end
+        currency_column = currency_column.to_s.freeze
```

Comment:
nit: use `# frozen_string_literal: true` at the top of the file instead of `.freeze`